.gitignore

```diff
@@ -1,6 +1,7 @@
 *.log
 .DS_Store
 node_modules
+coverage
```

Comment:
Since we aren't testing coverage right now... let's leave this out of this commit and add it separately... =)

Reply:
Here I just put the git to ignore the folder coverage. Do you think it is really necessary to undo this change?